src/librustc_data_structures/flock.rs

```diff
@@ -7,18 +7,17 @@
 #![allow(non_camel_case_types)]
 #![allow(nonstandard_style)]
 
+use std::fs::{File, OpenOptions};
 use std::io;
 use std::path::Path;
 
 cfg_if! {
-    if #[cfg(unix)] {
-        use std::ffi::{CString, OsStr};
-        use std::mem;
+    if #[cfg(target_os = "linux")] {
```

Comment:
Please add a comment here about WSL, so we remember why Linux is using something different. Then I think we're good to go!